sdk/core/azure-core-tracing-opentelemetry/README.md

```diff
@@ -0,0 +1,73 @@
+
+
+# Azure Core Tracing OpenTelemetry client library for Python
+
+## Getting started
+
+Install the opentelemetry python for Python with [pip](https://pypi.org/project/pip/):
+
+```bash
+pip install azure-core-tracing-opentelemetry --pre
+```
+
+Now you can use opentelemetry for Python as usual with any SDKs that is compatible
+with azure-core tracing. This includes (not exhaustive list), azure-storage-blob, azure-keyvault-secrets, azure-eventhub, etc.
+
+## Key concepts
+
+* You don't need to pass any context, SDK will get it for you
+
+## Examples
+
+There is no explicit context to pass, you just create your usual opentelemetry tracer and
+call any SDK code that is compatible with azure-core tracing. This is an example
+using Azure Monitor exporter, but you can use any exporter (Zipkin, etc.).
+
+```python
+from opentelemetry.ext.azure_monitor import AzureMonitorSpanExporter
+
+from opentelemetry import trace
+from opentelemetry.sdk.trace import Tracer
+
+from azure.core.settings import settings
+from azure.core.tracing.ext.opentelemetry_span import OpenTelemetrySpan
+
+from azure.storage.blob import BlobServiceClient
+
+# Declare that you want to trace Azure SDK with OpenTelemetry
+settings.tracing_implementation = OpenTelemetrySpan
+
+exporter = AzureMonitorSpanExporter(
+    instrumentation_key="uuid of the instrumentation key (see your Azure Monitor account)"
+)
+
+trace.set_preferred_tracer_implementation(lambda T: Tracer())
+tracer = trace.tracer()
+tracer.add_span_processor(
+    SimpleExportSpanProcessor(exporter)
+)
+
+with tracer.start_as_current_span(name="MyApplication"):
+    client = BlobServiceClient.from_connection_string('connectionstring')
+    client.delete_container('mycontainer')  # Call will be traced
```

Comment:
I would use `create_container` (generally, I go for less destructive examples if I have a choice :))

Reply:
fixed